File: src/Left Beehind/leftbeehind.cpp
#include<iostream>
using namespace std;

int main()
{
    int x, y;
    bool flag = true;
    while(flag) {
        cin >> x >> y;
        if(x==0 && y==0) {
            break;
        } else if (x + y == 13) {
            cout << "Never speak again." << endl;
        } else if (x == y) {
            cout << "Undecided." << endl;
        } else if (x > y) {
            cout << "To the convention." << endl;
        } else if (x < y) {
            cout << "Left beehind." << endl;
        }
    }
}

File: source/Left Beehind/leftbeehind.py
flag = False
while flag == False:
    x, y = map(int, input().split())
    if (x==0 and y==0):
        flag = True
    elif (x == y):
        print('Undecided.')
    elif (x + y == 13):
        print('Never speak again.')
    elif (x < y):
        print('Left beehind.')
    elif (x > y):
        print('To the convention.')